### PR TITLE
E2E default payment method tests

### PR DESCRIPTION
**Pull request by ndukatz**
## Description
New Tests for switching the default payment method

**What does this PR do?**
mocks all data, one test starting default cc then switching to gpay, the other is the opposite.

## How to test
- `yarn up` in one terminal
- `yarn cy:e2e -s ./cypress/integration/account/default-payment-method.spec.ts` in another
- Result should be "✔  All specs passed!"

## Make sure your .env contains:
```
REACT_APP_LOGIN_ROOT
REACT_APP_API_ROOT
REACT_APP_APP_ROOT
REACT_APP_LAUNCH_DARKLY_ID
REACT_APP_CLIENT_ID
MANAGER_OAUTH
```

### CODE REVIEW

File: packages/manager/cypress/integration/account/defalut-payment-method.spec.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      created: '2021-08-04T18:29:01',
+      data: { card_type: 'Visa', last_four: '2045', expiry: '07/2025' },
+    },
+  ],
+  page: 1,
+  pages: 1,
+  results: 2,
+};
+
+const getPaymentMethodDataDefaultCard = {
```
Comment (patthiel) [inline packages/manager/cypress/integration/account/defalut-payment-method.spec.ts:35]: this is a little redundant, we could create a new object based off the original, then change the defaults

something like

```javascript
let getPaymentMethodDataDefaultCard = { ... getPaymentMethodDataDefaultGpay };
// then set the values we care about
```

**Review by patthiel — APPROVED** [review]
one minor bit of feedback on redundancy, otherwise this is good! nice solve for the mock  issue.

**Comment by bnussman** [comment]
Agreed, use a Factory in the final draft

**Comment by bnussman** [comment]
Looking at the code right now, but can we fix the spelling of default in the file name `defalut-payment-method.spec.ts`?

**Review by bnussman — APPROVED** [review]

**Pull request merged**